pyoxidizer/src/projectmgmt.rs

```diff
@@ -41,7 +41,11 @@ use {
 pub fn default_target() -> Result<String> {
     // TODO derive these more intelligently.
     if cfg!(target_os = "linux") {
-        Ok("x86_64-unknown-linux-gnu".to_string())
+        if cfg!(target_arch = "aarch64") {
+            Ok("aarch64-unknown-linux-gnu".to_string())
```

Comment:
This change is correct. However, since we don't yet have default distributions for `aarch64-unknown-linux-gnu` in PyOxidizer, this will result in (cross-compile) builds breaking on aarch64-unknown-linux-gnu today. This is probably acceptable, as I don't think things work very well on aarch64-unknown-linux-gnu today.

Getting Linux aarch64 working is something I'd like to do for the next release. It was hung up behind some issues with python-build-standalone, which should hopefully be resolved with the next refresh of default Python distributions.

Reply:
While the default doesn't work, compiling the `python-build-standalone` repo for aarch64 works and manually setting the `PythonDistribution(local_path=...)` to the resulting file worked for me on an aarch64 machine (at least as far as opening an interpreter), so I don't think you are too far from everything just working :slightly_smiling_face:. I didn't try cross-compiling, just compiling it on an aarch64 host